### ISSUE TITLE
Earnings value is calculated incorrectly.

### ISSUE THREAD

**Issue by Scrivener07** [Scrivener07/FO4_Games#96]

**Comment by Scrivener07** [comment]
Im working on some changes. Here are the numbers so far. Looking good on bug fixes branch for this issue.

```
Turn    Scoring     Bet     Caps    Earnings
0       NAN         NAN     100     NAN
1       Loser       5       95      -5
2       Loser       5       90      -10
3       Loser       5       85      -15
4       Loser       5       80      -20
5       Loser       5       75      -25
6       Winner      5       80      -20
7       Winner      5       85      -15
8       Winner      5       90      -10
9       Loser       5       85      -15
10      Winner      5       90      -10
```

**Comment by Scrivener07** [comment]
Closed with pull request #97
